doc/btp_pbp.txt

```diff
@@ -0,0 +1,77 @@
+PBP Service (ID 30)
+=====================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02: Set Public Broadcast Announcement
+
+		Controller Index:	<controller id>
+		Command parameters:	Features (1 octet)
+							Metadata LTVs len (1 octet)
```

Comment:
I wonder if those should be passed by upper tester,  what exactly is pass here?  eg name is configured with separate command

Reply:
in Set Public Broadcast Announcement, the service data for ORG_BLUETOOTH_SERVICE_PUBLIC_BROADCAST_ANNOUNCEMENT is set. It contains metadata.

In Set Broadcast Name, ad type BLUETOOTH_DATA_TYPE_BROADCAST_NAME is set with the name, ie. the name is not in the Set Public Broadcast Announcement meta data.

That being set, it looks like we never call Set Broadcast Name and it's not set by default in our tester implementation. As it is an official field, I would suggest to keep it and improve the documentation a bit if that's ok with you.